uniprot-search/src/main/java/org/uniprot/store/search/document/uniprot/UniProtDocument.java

```diff
@@ -612,6 +615,7 @@ public int hashCode() {
                 computationalPubmedIds,
                 communityPubmedIds,
                 uniparc,
+                uniparcDeleted,
                 rheaIds,
                 suggests);
```

Comment:
We should use uniParc to be consistent and if possible we could also fix uniprot. It should be uniProt.

Reply:
deletedEntryUniParc